src/test/resources/wiremock/mappings/claim-email-and-phone-mismatch.json

```diff
@@ -0,0 +1,16 @@
+{
+  "claimStatus": "NEW",
+  "eligibilityStatus": "ELIGIBLE",
+  "verificationResult": {
+    "identityOutcome": "matched",
+    "eligibilityOutcome": "confirmed",
+    "addressLine1Match": "matched",
+    "postcodeMatch": "matched",
+    "isPregnantOrAtLeast1ChildMatched": true,
+    "mobilePhoneMatch": "not_matched",
+    "emailAddressMatch": "not_matched",
+    "qualifyingBenefits": "universal_credit",
+    "pregnantChildDOBMatch": "not_set",
```

Comment:
Shouldn't affect the test, but this should be `not_supplied` rather than `not_set`. Same in the other two JSON files too.

Reply:
That is right, I've updated to not_supplied